data/ecosystem/adopters.yaml

```diff
@@ -19,6 +24,11 @@
   components: [Ruby]
   reference: ''
   contact: ''
+- name: Dyte
+  url: https://dyte.io
+  components: [Golang, Javascript]
+  reference: 'https://dyte.io/blog/opentelemetry-at-dyte-part-i/'
+  contact: ''
```

Comment:
Can you put a contact here, this is for us (maintainers) to know whom to reach if for example that link expires, can be your github handle (preferred) or an email address